indico/modules/events/surveys/fields/choices.py

```diff
@@ -29,6 +30,24 @@
 from indico.web.forms.validators import HiddenUnless
 
 
+class OptionIDMixin(object):
+
+    def copy_field_data(self):
+        """Return a copy of the field's configuration data without the IDs used to identify answer options."""
+        field_data_copy = super(OptionIDMixin, self).copy_field_data()
+        for option in field_data_copy['options']:
+            del option['id']
+        return field_data_copy
+
+    @staticmethod
+    def process_imported_data(data):
+        """Generate the options' IDs"""
+        if 'options' in data:
```

Comment:
`data = deepcopy(data)` - since you return the updated data I think it's cleaner to not modify the dict passed to this method